sysutils/gptfdisk/Portfile

```diff
@@ -17,17 +17,22 @@ long_description        GPT fdisk (gdisk) is a disk partitioning tool loosely mo
 homepage                http://www.rodsbooks.com/gdisk/
 master_sites            sourceforge:project/gptfdisk/gptfdisk/${version}
 
-checksums               md5     d7f3d306b083123bcc6f5941efade586 \
-                        sha1    ad28c511c642235815b83fffddf728c117057cba \
-                        rmd160  21bef329361690a1a7d44e464eb9d682c6335137 \
-                        sha256  864c8aee2efdda50346804d7e6230407d5f42a8ae754df70404dd8b2fdfaeac7
+
+checksums               sha256 89fd5aec35c409d610a36cb49c65b442058565ed84042f767bba614b8fc91b5c \
+                        rmd160 9bf949224cb5fa4fe7df4f332c22a7106a2cfa0c
```

Comment:
As upstream on sf.net only publishes sha1/md5 checksums, listing these helps to verify that the correct file was downloaded.